packages/arsenalimagemounter.vm/arsenalimagemounter.vm.nuspec

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>arsenalimagemounter.vm</id>
+    <version>3.10.262</version>
+    <authors>Arsenal Recon</authors>
+    <description>Mounts the contents of disk images as complete disks in Windows.</description>
+    <dependencies>
+      <dependency id="common.vm" />
+      <dependency id="dotnet-6.0-desktopruntime" version="[6.0.25]" />
```

Comment:
@naacbin could you add a metapackage for this dependency as mentioned in https://github.com/mandiant/VM-Packages/pull/778/files#r1429841023? This PR is smaller so it will be easier to review it and then use it in the other PR that still has some discussion.

Reply:
I think we can merge this PR without addressing this, as until we merge #778 there is this package using this dependency. The metapackage can be added as an independent PR (before #778) or together with #778